tools/cockpit.spec

```diff
@@ -414,10 +452,15 @@ authentication via sssd/FreeIPA.
 %doc %{_mandir}/man8/remotectl.8.gz
 %doc %{_mandir}/man8/pam_cockpit_cert.8.gz
 %doc %{_mandir}/man8/pam_ssh_add.8.gz
+%dir %{_sysconfdir}/cockpit
 %config(noreplace) %{_sysconfdir}/cockpit/ws-certs.d
 %config(noreplace) %{_sysconfdir}/pam.d/cockpit
 %config %{_sysconfdir}/issue.d/cockpit.issue
+%dir %{_sysconfdir}/motd.d
```

Comment:
This feels wrong.  /etc/motd.d is owned by `setup` and `pam`, I don't think it's cockpit-ws'es business to provide/ship it. Isn't that directory owned by anything in OpenSUSE?

Reply:
probably. I could move that to the file system package indeed. Will do.

Reply:
Oh, but moving the motd.d link to /usr/lib/motd.d/ might help with *this* part, of course.

Reply:
probably requires a newer pam_motd than what's in stable releases you may want to still support though